kernel-headers/rdma/efa-abi.h

```diff
@@ -85,11 +85,17 @@ enum {
 	EFA_QP_DRIVER_TYPE_SRD = 0,
 };
 
+enum {
```

Comment:
Please use real SHA and not ?? when creating this patch.
The kernel headers are pushed to wip/leon-for-next

Reply:
Thanks, updated the commit.